src/documentation/Search.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Box,
+  Divider,
+  IconButton,
+  Input,
+  InputGroup,
+  InputLeftElement,
+  InputRightElement,
+  Stack,
+  Text,
+} from "@chakra-ui/react";
+import { useCallback, useRef } from "react";
+import { RiCloseLine, RiSearch2Line } from "react-icons/ri";
+import { SearchResults } from "./search-hooks";
+import SearchResultList from "./SearchResultList";
+
+interface SearchProps {
+  onClose: () => void;
+  results: SearchResults | undefined;
+  setResults: React.Dispatch<React.SetStateAction<SearchResults | undefined>>;
+  query: string;
+  setQuery: React.Dispatch<React.SetStateAction<string>>;
+  handleQueryChange: React.ChangeEventHandler<HTMLInputElement>;
```

Comment:
onQueryChange

Reply:
[`975574f`](https://github.com/microbit-foundation/python-editor-next/pull/452/commits/975574fa3bd9e9972865ac0bf35c05054b7dcaf1)

Reply:
tweaked this to onQueryChange rather than onHandleQueryChange